### ISSUE TITLE
performance metric

### ISSUE THREAD

**Issue by chengsoonong** [chengsoonong/mclass-sky#84]
compare:
http://scikit-learn.org/stable/modules/generated/sklearn.metrics.auc.html
with balanced posterior accuracy.

**Comment by chengsoonong** [comment]
Write a section in your report describing this and the results.
